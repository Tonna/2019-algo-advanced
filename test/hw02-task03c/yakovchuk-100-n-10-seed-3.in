10
2 6 7 1 10 5 8 3 9 4
