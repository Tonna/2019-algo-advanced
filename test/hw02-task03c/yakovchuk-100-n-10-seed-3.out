4 4 8 10 9 8 7 10 9 10
